kibbeh/src/modules/room/UserPreviewModal.tsx

```diff
@@ -71,7 +71,11 @@ const UserPreview: React.FC<{
     return <div className={`flex text-primary-100`}>This user is gone.</div>;
   }
 
-  const canDoModStuffOnThisUser = !isMe && (iAmCreator || iAmMod) && !isCreator;
```

Comment:
Can we change name on this variable to something more prof=

Reply:
dogeperms or modperms your choice